.gitignore

```diff
@@ -2,3 +2,4 @@
 addons/sourcemod/scripting/.atom-build.json
 .vscode/
 addons/sourcemod/scripting/tests/logTest.sp
+addons/sourcemod/scripting/tests/ttt_inventory_test.sp
```

Comment:
You should keep it for us or merge it into ttt_api_test.

Reply:
Oof, this was a mistake, I never meant for anyone to get a whiff of this.